Detections/ProofpointTAP/MalwareLinkClicked.yaml

```diff
@@ -0,0 +1,25 @@
+id: 8675dd7a-795e-4d56-a79c-fc848c5ee61c
+name: Malware Link Clicked
+description: |
+  'Creates an incident in the event a user clicks on an email link that is classified as a malware.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CustomConnector
```

Comment:
Same as previous for the connectorId and dataTypes

Reply:
corrections have been made and pushed.